typescript/packages/plugins/erc20/src/erc20.service.ts

```diff
@@ -22,11 +22,17 @@ export class Erc20Service {
     }
 
     @Tool({
-        description: "Get the ERC20 token info by its symbol, including the contract address, decimals, and name",
+        description:
```

Comment:
hmm no need for all these changes. 

btw, we will move the resolve address part to its own tool so no worries this will have its own PR :)

Reply:
Ser @0xaguspunk these are done by my formatter automatically. Could you suggest what to do for TS files?

Reply:
Check the new comments! :)